irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -111,6 +112,9 @@ namespace iroha {
        */
       void tryRollback(soci::session &session);
 
+      std::optional<std::shared_ptr<iroha::ametsuchi::PostgresBurrowStorage>>burrow_storage_;
+      std::optional<std::shared_ptr<soci::session>> sql;
```

Comment:
according to convention should be:
```suggestion
      std::optional<std::shared_ptr<soci::session>> sql_;
```